Makefile

```diff
@@ -95,17 +95,22 @@ hugo/run:
 run: docker/build
 	$(DOCKER_RUN) hugo/run
 
+## TODO change /static to /public
 ## Build customized utterances widget
 utterances/build:
 	rm -rf utterances
 	git clone --branch feature/integration https://github.com/cloudposse/utterances.git
 	cd utterances && yarn && yarn build
-	mv utterances/dist/*.js public/js
-	mv utterances/dist/*.css public/css
-	mv utterances/dist/*.png public/assets
-	mv utterances/dist/*.svg public/assets
-	mkdir -p public/utterances/
-	mv utterances/dist/utterances.html public/utterances/index.html
+	mkdir -p static/js/utterances static/css/utterances static/assets/utterances
```

Comment:
We don't have to be so strict about adhering to `static/js`, `static/css`, etc when it relates to utterances. I would be okay with just deploying it all to `static/utterances` since it's a self-contained app